lib/msal-node/src/request/InteractiveRequest.ts

```diff
@@ -3,11 +3,13 @@
  * Licensed under the MIT License.
  */
 
+import { ILoopbackClient } from "../network/ILoopbackClient";
 import { AuthorizationUrlRequest } from "./AuthorizationUrlRequest";
 
 export type InteractiveRequest = Pick<AuthorizationUrlRequest, "authority"|"correlationId"|"claims"|"azureCloudOptions"|"account"|"extraQueryParameters"|"tokenQueryParameters"|"extraScopesToConsent"|"loginHint"|"prompt"> & {
     openBrowser: (url: string) => Promise<void>;
     scopes?: Array<string>;
     successTemplate?: string;
     errorTemplate?: string;
+    customLoopbackClient?: ILoopbackClient
```

Comment:
Let's just call this loopbackClient
```suggestion
    loopbackClient?: ILoopbackClient
```

Reply:
done!